.buildkite/pipeline_cross.py

```diff
@@ -38,7 +38,7 @@ def restore_step(label, src_instance, src_kv, dst_instance, dst_os, dst_kv):
 def cross_steps():
     """Generate group steps"""
     instances_x86_64 = ["c5n.metal", "m5n.metal", "m6i.metal", "m6a.metal"]
-    instances_aarch64 = ["m6g.metal", "m7g.metal"]
+    instances_aarch64 = ["m7g.metal"]
```

Comment:
nit: it might be helpful to have explanation why m6g.metal doesn't work.

Reply:
Will do in follow up